app/products/playbooks/components/progress_bar/progress_bar.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useMemo} from 'react';
+import {View, type ViewStyle, type StyleProp} from 'react-native';
+
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme) => ({
+    progressBar: {
+        position: 'absolute',
+        bottom: 0,
+        left: 0,
+        height: 2,
+        backgroundColor: theme.onlineIndicator,
+        borderRadius: 2,
+    },
+}));
+
+type Props = {
+    progress: number;
+    isActive: boolean;
+    testID?: string;
+}
+
+function ProgressBar({progress, isActive, testID}: Props) {
```

Comment:
nitpick: this shoudl be a component of the app, not of the product, seems such a basic component.

Reply:
This is the particular way playbooks handle progress bars (the green line below a playbook / checklist). We don't use it anywhere else in the app, but if we start using something similar, I agree with you.